components/VideoEmbedPreview/VideoEmbedPreview.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { Meta } from "@storybook/react";
+import VideoEmbedPreview from "./VideoEmbedPreview";
+
+export default {
+  title: "VideoEmbedPreview",
+  component: VideoEmbedPreview,
+  args: {
+    video: {
+      url: "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+    },
+  },
+  parameters: {
+    nextjs: {
+      appDirectory: true,
+    },
+  },
+} as Meta<typeof VideoEmbedPreview>;
```

Comment:
Instead of `as` let's do `satisfies` and assign this object to a variable, say `meta` 
Then, we `export default` the variable `meta`. It should look something like this:
```ts
const meta = {
  // ...
} satisfies Meta<typeof /* Component */ >

export default meta
```
That way, we can reuse `meta` in our stories. Like so:
```ts
type Story = StoryObj<typeof meta> 

export const Default: Story = {}
```

---

If you want, you can read more on "[How to write stories](https://storybook.js.org/docs/react/writing-stories/introduction)"

Reply:
We also need this for the `Hero` component (#284)